lib/order_dispatch.rb

```diff
@@ -0,0 +1,23 @@
+# require_relative 'takeaway'
+require_relative 'order'
+require 'dotenv'
+require 'twilio-ruby'
+
+class OrderDispatch
+  def initialize(client_klass = Twilio::REST::Client)
+    @credentials = Dotenv.load
+    @client      = client_klass.new(
+      @credentials['account_sid'], @credentials['auth_token']
+    )
+  end
+
+  def new_order(order)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `order`. If it's necessary, use `_` or `_order` as an argument name to indicate that it won't be used. You can also write as `new_order(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)